sentences/de/_common.yaml

```diff
@@ -64,5 +65,7 @@ expansion_rules:
   alle_luefter: "alle (Ventilatoren | Lüfter)"
   brightness: "{brightness:brightness_pct} [Prozent]"
   temperature: "{temperature} [Grad] [{temperature_unit}]"
+  lampe: "([(der | die | das | des | den)] (Lampe |  Licht[s] | Spot[s] | Strahler[s]))"
```

Comment:
While the above genders/cases are gramatically correct, would it be worth including others to allow for children and non-native speakers who make mistakes?